schema/defs-linux.json

```diff
@@ -194,7 +194,8 @@
                     "type": "boolean"
                 },
                 "type": {
-                    "type": "string"
+                    "type": "string",
+                    "pattern": "^[acb]$"
```

Comment:
This is not correct, you can have empty `type` fields. It should be `^[acb]?$`.

Reply:
> It should be `^[acb]?$`.

No, it's right without the `?`.  Empty-string values are not valid in the JSON.  This property can be *unset* in JSON, which we cover inthe JSON Schema by [not listing `type` in `required`][1].  We test the unset-`type` case with [this][2], via [this][3] and [this][4].

In the non-pointer Go property, "unset" is not an option, but the `omitempty` means empty-string Go values are treated as unset for JSON Schema validation.

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0/schema/defs-linux.json#L224
[2]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0/schema/test/config/good/spec-example.json#L257
[3]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0/schema/Makefile#L21
[4]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0/.travis.yml#L25